trace/kafka/kafka.go

```diff
@@ -86,7 +89,7 @@ func NewAsyncProducer(brokers []string, oo ...OptionFunc) (*AsyncProducer, error
 
 	prod, err := sarama.NewAsyncProducer(brokers, ap.cfg)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create sync producer: %w", err)
+		return nil, errors.Wrap(err, "failed to create async producer")
```

Comment:
why use Wrap ? was Errorf not good enough ?

Reply:
I did actually removed patron's error.Wrap in favor of fmt...

Reply:
It was the only use of `fmt` in the package; I changed it in order to remove the dependency, but I'll restore it back!